lib/rucio/core/replica.py

```diff
@@ -799,6 +803,37 @@ def _list_replicas_for_files(file_clause, state_clause, files, rse_clause, updat
             {'scope': scope, 'name': name} in files and files.remove({'scope': scope, 'name': name})
 
 
+def get_multi_cache_prefix(cache_site, filename):
+
+    # print('Looking up prefix for cache:', cache_site, 'and filename:', filename)
+    x_caches = read_from_cache('CacheSites', expiration_time=60)
+    if not x_caches:
+        # print('reloading xcache sites and ranges.')
+        vp_endpoint = 'https://vps.cern.ch/serverRanges'
```

Comment:
this would run any test of this functionality against a live instance? Where is the documentation on this endpoint located? This will probably not work for organizations using Rucio outside of CERN.

Reply:
Yes. Once we have all of VP integrated in Rucio this won't be needed. But for now even someone testing from outside won't see any issues.

Reply:
Yes, just put it in the config table aswell. 
E.g.
```
[virtualplacement]
vp_endpoint= https://vps.cern.ch/serverRanges
```